app/Shiny_AFS_final_app3.R

```diff
@@ -0,0 +1,834 @@
+library(shiny)
+library(shinyWidgets)
+library(tidyverse)
+library(sf)
+library(leaflet)
+library(DT)
```

Comment:
Missing this library for the `popupTable` function. 

```suggestion
library(DT)
library(leafpop)
```